Pod/Classes/URBNArrayDataSourceAdapter.m

```diff
@@ -260,6 +281,21 @@ - (void)removeItemAtIndexPath:(NSIndexPath *)indexPath {
     }
 }
 
+- (void)removeLastSection {
+    if ([self isSectioned] && (self.items.count > 0)) {
+        [self.items removeLastObject];
+
+        NSIndexSet *deletedSection = [NSIndexSet indexSetWithIndex:self.items.count];
```

Comment:
Also i like how you put this outside the condition. Why not do the same with the append method